qa/rpc-tests/test_framework/script.py

```diff
@@ -36,7 +37,10 @@
 OPCODE_NAMES = {}
 
 def hash160(s):
-    return hashlib.new('ripemd160', sha256(s)).digest()
+    return RIPEMD160.new(sha256(s)).digest()
+
+# def hash160(s):
+#     return hashlib.new('ripemd160', sha256(s)).digest()
```

Comment:
Even though it's not strictly necessary, I would definitely remove the old code (not sure if it's also possible to remove the import of `hashlib`).